apps/web/data/activities.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+        "tittle": "Mini sessão de abertura",
+        "details": "10:00 - 10:30 || Júlio Pinto - Organizador DojoCon",
+        "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer eros nisi, blandit id justo eget, auctor tincidunt est."
```

Comment:
About stuff like this just take out the descrpition, in most of them, the exception being CoderDojo Presentation, Sumo Hacking Challenge, JNO Talk, Quiz and Champions Roundtable

Reply:
I disagree in this one, once that it's important to give a little context to the people interested. It intends to be something short and clear... :eyes:

Reply:
In mobile it even disappears :sweat_smile: